marketdata/rest.go

```diff
@@ -983,6 +1084,26 @@ func (c *Client) cryptoFeed(fromReq string) string {
 	return "us"
 }
 
+func (c *Client) cryptoURL(fromReq request) string {
+	return fmt.Sprintf("%s/%s/%s", c.opts.BaseURL, func(p bool) string {
+		if !p {
+			return cryptoPrefix
+		}
+		return cryptoPerpPrefix
+	}(fromReq.isPerp()), func(fromReq request) string {
+		if fromReq.cryptoFeed() != "" {
+			return fromReq.cryptoFeed()
+		}
+		if c.opts.CryptoFeed != "" {
+			return c.opts.CryptoFeed
+		}
+		if fromReq.isPerp() {
+			return "global"
+		}
+		return "us"
+	}(fromReq))
```

Comment:
This feels a bit too functional for me in Go :P

```suggestion
	prefix := cryptoPrefix
	if fromReq.isPerp() {
		prefix = cryptoPerpPrefix
	}
	feed := fromReq.cryptoFeed()
	if feed == "" {
		if fromReq.isPerp() {
			feed = GLOBAL
		} else {
			feed = US
		}
	}
	return fmt.Sprintf("%s/%s/%s", c.opts.BaseURL, prefix, feed)
```

Reply:
Added the suggested change.